### ISSUE TITLE
Metadata is included in message when using Winston 3

### ISSUE THREAD

**Issue by padiazg** [winstonjs/winston-mongodb#122]
Just started to use Winston 3.0-rc3 + winston-mongodb 4.0.0-rc1 in a new project, and found that metadata is writed correctly in the documents, but also they are included in the message field.

I have this initialization code:
``` javascript
const { createLogger, format, transports } = require('winston');
const { useMongodb, mongodbUrl } = require('./config');

const logger = createLogger({
    level: 'info',
    format: format.json(),
    transports: [
        new transports.Console({ format: format.simple(), level: 'error' }),
    ]
}); // logger ... 

if (useMongodb) {
    require('winston-mongodb').MongoDB;
    logger.add(
        new transports.MongoDB({
            db: mongodbUrl,
            collection: 'logs',
            level: 'info',
            format: format.simple(),
            label: false
        }) // new transports.MongoDB ...
    ); // logger.add ...
} // if (useMongodb) ...
```
then for a test put this after calling the initialization code, note the new recomended format.
``` javascript
logger.info({message: 'logger0', level: 'info', meta:{ k0:'abcdef', k: 12345 }});
```
and this is what is saved in mongodb
``` JSON
{
   "_id":"5ab66d72020d5306d0719b52",
   "timestamp":"2018-03-24T15:23:30.007Z",
   "level":"info",
   "message":"info: logger0 {\"meta\":{\"k0\":\"abcdef\",\"k\":12345}}",
   "meta":{
      "k0":"abcdef",
      "k1":12345
   }
}
```
Doing a littlle of debugging found the line 191 on winston-mongodb.js
```javascript
let message = util.format(info[MESSAGE], ...(info.splat || []));
```
console.log(info[MESSAGE]) throws:
```
info: logger0 {"meta":{"k0":"abcdef","k":12345}}
```
and console.log(info) throws:
```
{ message: 'logger0',
  level: 'info',
  meta: { k0: 'abcdef', k: 12345 },
  [Symbol(level)]: 'info',
  [Symbol(message)]: 'info: logger0 {"meta":{"k0":"abcdef","k":12345}}' }
```
so, replacing line 191 for
``` javascript
// let message = util.format(info[MESSAGE], ...(info.splat || []));
let message = util.format(info.message, ...(info.splat || []));
```
it saves right as needed:
``` JSON
{
   "_id":"5ab688c48fd9cf5ac0105494",
   "timestamp":"2018-03-24T17:20:04.747Z",
   "level":"info",
   "message":"logger0",
   "meta":{
      "k0":"abcdef",
      "k":12345
   }
}
```
Question is, is this a bug?, or I'm missing something at initializing winston or winston-mongodb?
If it's is a bug, should I make a pull request? (this later wold be great for me)

Best regards,

**Comment by cyrus-zighra** [comment]
see [https://github.com/winstonjs/winston/issues/1285](https://github.com/winstonjs/winston/issues/1285)

**Comment by yurijmikhalevich** [comment]
fixed in version 4.0.2.
